### PR TITLE
Fix update script so it also works on mac.

### PR DESCRIPTION
**Pull request by ray-roestenburg-da**
This is followed up by https://github.com/digital-asset/decentralized-canton-sync/pull/6

### CODE REVIEW

File: scripts/update.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ function copy_daml {
     done
 }
 
-pkgs=$(ls -I "dars*" -I "splitwell*" $CN_REPO_ROOT/daml)
+pkgs=$(ls $CN_REPO_ROOT/daml | grep -v "dars*" | grep -v "splitwell*")
```
Comment (stas-sbi) [inline scripts/update.sh:22]: grep takes regular expressions (BRE by default), so `dars*` would match `dar`, `dars`, `darss` and so on

Comment (stas-sbi) [inline scripts/update.sh:22]: My guess is what you want is:
```suggestion
pkgs=$(ls $CN_REPO_ROOT/daml | grep -v "^\(dars\|splitwell\).*")
```

**Comment by ray-roestenburg-da** [comment]
Thanks @stas-sbi

**Review by stas-sbi — APPROVED** [review]

**Review by isegall-da — APPROVED** [review]
Thanks

**Review by cocreature — APPROVED** [review]

**Pull request merged**